asb-native/src/main/java/org/ballerinax/asb/util/ModuleUtils.java

```diff
@@ -39,6 +40,18 @@ public static Module getModule() {
     }
 
     public static void setModule(Environment env) {
+        String logLevel = "";
+        try {
+            logLevel = System.getenv("ASB_CLOUD_LOGS");
+        } catch (Exception e) {
+            // If a security manager exists, its checkPermission method is called with a
```

Comment:
Do we need to do something here?

Reply:
Added ballerina error to the module initialization.